README.md

```diff
@@ -101,6 +101,10 @@ meson _build --prefix=/usr/local
 ninja -C _build install
 ```
 
+## Link to the guide to enable the podman socket
+
+[Podman socket activation](https://github.com/containers/podman/blob/cea9340242f3f6cf41f20fb0b6239aa3db5decd6/docs/tutorials/socket_activation.md)
```

Comment:
I'd like to have a bit more context here. The user who reads it should get an understanding why this is important.
I suggest writing something similar as in the lightbulb popup


> To connect to the local Podman instance, the systemd socket must be enabled. You can enable it permanently by issuing the following command:
> ```
> systemctl --user enable --now podman.socket
> ```
> Visit the <a href="https://github.com/containers/podman/blob/cea9340242f3f6cf41f20fb0b6239aa3db5decd6/docs/tutorials/socket_activation.md">official documentation</a> for more information.